api/migrations/0003_auto_20150218_1132.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
```

Comment:
And like the other one this one is also not needed. Can you remove them both please? 

http://stackoverflow.com/questions/9498201/delete-file-from-pull-request-on-github